apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/PartitionedReadTests.cs

```diff
@@ -53,7 +65,9 @@ public async Task DistributedReadAsync(bool query)
                 {
                     transaction.DisposeBehavior = DisposeBehavior.CloseResources;
                     cmd.Transaction = transaction;
-                    var partitions = await cmd.GetReaderPartitionsAsync(1000);
+                    var partitions = dataBoostEnabled == null ?
+                        await cmd.GetReaderPartitionsAsync(1000) :
+                        await cmd.GetReaderPartitionsAsync(PartitionOptions.Default.WithPartitionSizeBytes(1000).WithDataBoostEnabled(dataBoostEnabled));
```

Comment:
We need to add some asserts to this test (and once you do that, I'm not certain that reusing the same method is a good idea re clarity, maybe copy paste the common bits).

- We need to assert that the requests in the CommandPartitions returned in line 68 have DataBoostEnabled set to the correct value.
- We also need to assert that after serialization/deserialization the CommandPartitions built in line 75 have DataBostEnabled set to the correct value, possibly assert this before using the partitions in DistributedReadWorkerAsync.